src/app/component/general/eco-news/news-list/news-list.component.html

```diff
@@ -0,0 +1,20 @@
+<div class="wrapper">
+  <app-remaining-count [remainingCount]="remaining"></app-remaining-count>
+  <app-change-view-button (view)="chageView($event)"></app-change-view-button>
+</div>
+
+<div>
+  <ul class="list" [ngClass]="{ 'gallery-view-active': view }">
+    <li [ngClass]="{
```

Comment:
bad formatting, attributes should be in one column

Reply:
Fixed